pkg/scheduler/api/job_info.go

```diff
@@ -38,7 +38,12 @@ type TaskInfo struct {
 	Name      string
 	Namespace string
 
-	Resreq *Resource
+	// Resreq is pod resource request, and it does not contain init containers resource request.
+	// ResreqWithInitContainers is also pod resource request, however it contains init containers resource request.
+	// To be consistent with kubernetes default scheduler, ResreqWithInitContainers is only used for predicates
+	// of actions (e.g.allocate, backfill, preempt, reclaim), please use Resreq for other cases.
+	Resreq                   *Resource
+	ResreqWithInitContainers *Resource
```

Comment:
How about `InitResreq`? The `Resreq` is the resource that used when task running; and the `InitResreq` is the resource that used to launch a task.

Reply:
OK, it is shorter :) Addressed.